packages/ssx-server-middleware/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@spruceid/ssx-server-middleware",
```

Comment:
When debugging Next, ssx-server was pulling in Express. I thought pulling it out into it's own package would be good, but that was incorrect. I do think the effort done towards that though is good, and we should keep it as it's own package

Reply:
This point is open for discussion, we can keep it as a part of `ssx-server` if that's our consensus

Reply:
No problem for me